src/config.c

```diff
@@ -263,6 +263,14 @@ int yesnotoi(char *s) {
 }
 
 void appendServerSaveParams(time_t seconds, int changes) {
+    int i;
+    for (i = 0; i < server.saveparamslen; i++) {
+        if (server.saveparams[i].seconds == seconds) {
+            server.saveparams[i].changes = changes;
```

Comment:
i think we need to do that only if the new `changes` is smaller than the orig.

this way we're backwards compatible with bad configurations